packages/components/src/gradient-picker/index.tsx

```diff
@@ -159,7 +210,6 @@ export default function GradientPicker( {
 				{ ( gradients?.length || clearable ) && (
 					<Component
 						className={ className }
-						clearable={ clearable }
```

Comment:
`clearable` is not used by either `SingleOrigin` or `MultipleOrigin`, and appears to have been accidentally passed down in https://github.com/WordPress/gutenberg/pull/35970 along with the rest of `GradientPicker`'s props.